# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/systemservices/CamerasChild.h
# Commit: d838dc7d2bb8
# Full Hash: d838dc7d2bb89ab5c9c2e8e93895db1c9310560f
# Author: Dan Minor <dminor@mozilla.com>
# Date: 2021-11-01 18:17:33
# Regressor Bug: 1654112
# File Overlap Count: 4
# Description:
#   Bug 1654112 - Update include paths for moved upstream code. r=ng
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D130059
# ==============================================================================

diff -r 587c40771588 -r d838dc7d2bb8 dom/media/systemservices/CamerasChild.h
--- a/dom/media/systemservices/CamerasChild.h	Fri Oct 23 13:49:51 2020 -0400
+++ b/dom/media/systemservices/CamerasChild.h	Fri Oct 23 14:02:54 2020 -0400
@@ -17,7 +17,7 @@
 
 // conflicts with #include of scoped_ptr.h
 #undef FF
-#include "webrtc/modules/video_capture/video_capture_defines.h"
+#include "modules/video_capture/video_capture_defines.h"
 
 namespace mozilla {
 